frontend/src/pull-card/commit-statuses.tsx

```diff
@@ -130,3 +133,14 @@ function CommitStatuses({pull}: {pull: Pull}) {
    </Popover>
    );
 });
+
+function Duration({status}: {status:CommitStatus}) {
+   const {started_at, completed_at} = status.data;
+   if (!started_at) {
+      return <>Queued</>;
+   }
+
+   const completed = completed_at || Date.now() / 1000;
+   const min = Math.ceil((completed - started_at)/60);
+   return min ? <>{min}m</> : null;
```

Comment:
Might consider moving the `chakra.span` into here, so we can skip rendering the `<span>` if it's going to be empty.

Reply:
Fixed.